tarot.py

```diff
@@ -0,0 +1,61 @@
+from os import listdir
+from random import shuffle, randint
+from PIL import Image
+from tempfile import TemporaryFile
+import itertools
+
+def main():
+    get_reading(78)
+
+def get_reading(amount):
+    # cards in resources folder
+    cards = listdir("resources/tarot")
+    # magic shuffling
+    shuffle(cards)
+    reading = []
+    for i in range(amount):
+        # how 2 reverse a queue
+        reading.append(cards.pop())
+
+    # return the tempfile with the image
+    return(make_image(reading))
+
+def make_image(reading):
+    reading_image = Image.new('RGB', (250 * len(reading), 429))
+
+    for i in range(len(reading)):
+        # chance for flipped card
+        if randint(0,10) == 0:
+            card_image = Image.open("resources/tarot/" + reading[i])
+            image_flipped = card_image.transpose(Image.FLIP_TOP_BOTTOM)
+            reading_image.paste(im=image_flipped, box=(250 * i, 0))
+        #normal card
+        else:
+            reading_image.paste(im=Image.open("resources/tarot/" + reading[i]), box=(250 * i, 0))
+
+    # do tempfile fuckenings to give pic to telegram somewhat safely
+    # doesn't work (on Windows) without making it delete=False and then killing it later.
+    # don't really like the fragmentation but can't figure out a better way.
+    fp = TemporaryFile(delete=False)
+    reading_image.save(fp, 'jpeg', quality=75)
+    return(fp)
+
+def explain_card(text, card_data):
```

Comment:
Should this tarot.py module be the one reading card_data from db and keeping it in memory?